src/controllers/handlers/livekit-webhook-handler.ts

```diff
@@ -22,6 +32,21 @@ export async function livekitWebhookHandler(
 
   const webhookEvent = await livekit.getWebhookEvent(body, authorization)
 
+  const dataWarehouseEvent = WEBHOOK_EVENTS_TO_SEND_TO_DATA_WAREHOUSE[webhookEvent.event]
+  if (dataWarehouseEvent) {
+    const { participant } = webhookEvent
+
+    setImmediate(async () => {
+      await dataWarehouseClient.sendEvent({
+        context: 'notification_server',
+        event: dataWarehouseEvent,
+        body: {
+          address: participant?.identity
+        }
+      })
+    })
```

Comment:
What do you think about returning?
```suggestion
  if (dataWarehouseEvent) {
    const { participant } = webhookEvent

    setImmediate(async () => {
      await dataWarehouseClient.sendEvent({
        context: 'notification_server',
        event: dataWarehouseEvent,
        body: {
          address: participant?.identity
        }
      })
    })
    return
```